examples/bri-3/src/bri/auth/capabilities/login/login.command.handler.ts

```diff
@@ -24,10 +24,18 @@ export class LoginCommandHandler implements ICommandHandler<LoginCommand> {
       throw new Error(errorMessage.USER_NOT_AUTHORIZED);
     }
 
-    const jwtPayload = {
-      username: bpiSubject.name,
-      sub: bpiSubject.id,
+    const serviceUrl = process.env.SERVICE_URL;
+    const subjectDid = `did:ethr:0x5:${publicKey}`;
```

Comment:
not the best code, just for illustration - should we send did or public key in login request? both should work i think

Reply:
did does not change ever, pub keys do. So let's use the did since that should be the userId in the bpiSubjectAccount

Reply:
@Therecanbeonlyone1969 could you please clarify this? currently bpiSubject contains publicKey and we are fetching bpiSubject here by public key - not sure i follow, should did replace publicKey in bpiSubject or something else? also there is no userId in bpiSubjectAccount, there are db foreign key relations with bpiSubject (ownerBpiSubjectId and creatorBpiSubjectId)

if we need some db changes too let's please do it in separate issue and merge current auth work so we dont have to maintain 3 PRs, we can build up on top of this